.github/workflows/push.yaml

```diff
@@ -212,8 +212,8 @@ jobs:
       - name: Helm Template
         run: |
           helm template reloader deployments/kubernetes/chart/reloader/ \
-            --set reloader.deployment.resources.limits.cpu=150m \
-            --set reloader.deployment.resources.limits.memory=512Mi \
+            --set-string reloader.deployment.resources.limits.cpu=1 \
+            --set reloader.deployment.resources.limits.memory=128Mi \
```

Comment:
Don't set the limits to 128Mi. In my previous comment, I was talking about requests.

So, requests at 128Mi and limits at 512Mi is good for defaults.

Reply:
Actually, no it is not. It is not recommended to have memory limits != request. Memory cannot be throttled or reclaimed in case of pressure the container will be restarted which most likely will destabilize the system even further.
Go, Java and other runtimes *WILL* use the entire memory available under limits causing unwelcome container restarts.